modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase

```diff
@@ -149,12 +149,58 @@ definition {
 	}
 
 	@description = "Verify that the users are able to identify the past stages that the workflow instance has already passed by"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanIdentifyPastStagesOnWorkflow {
 
-		// TODO LRQA-72393 CanIdentifyPastStagesOnWorkflow pending implementation
+		property portal.acceptance = "true";
+		property test.name.skip.portal.instance = "WorkflowMetricsAllItemsList#TooltipDisplaysBulletListWhenMultipleItems";
```

Comment:
The skip property it's misconfigured. Should be : 
`property test.name.skip.portal.instance = "WorkflowMetricsAllItemsList#CanIdentifyPastStagesOnWorkflow";`